server/app/views/admin/programs/ProgramBlockPredicateConfigureView.java

```diff
@@ -745,7 +745,17 @@ private DivTag createValueField(
                     "hidden",
                     "text-xs",
                     BaseStyles.FORM_LABEL_TEXT_COLOR)
-                .withText("Enter a list of comma-separated values. For example, \"v1,v2,v3\"."));
+                .withText("Enter a list of comma-separated values. For example, \"v1,v2,v3\"."))
```

Comment:
_very_ nit here and 756: This file seems to _mostly_ use `div("text").withClasses()` instead of `div().withClasses().withText()`